packages/wrangler/bin/wrangler.js

```diff
@@ -6,6 +6,7 @@ const os = require("os");
 const semiver = require("semiver");
 
 const MIN_NODE_VERSION = "16.7.0";
+const debug = process.env["WRANGLER_LOG"] === "debug" ? console.log : () => {};
```

Comment:
does console.log work when it's detached from `console`? I figure you've tested this, but just confirming. Test it on v16 as well